sentry_kube/bin/important-diffs-only

```diff
@@ -12,10 +12,12 @@ def main(from_dir, to_dir) -> None:
     from_dir_apply_results = important_diffs.process_folder(from_dir)
     to_dir_apply_results = important_diffs.process_folder(to_dir)
 
-    for warning_message in important_diffs.warn_user_for_changes(from_dir_apply_results, to_dir_apply_results):
+    for warning_message in important_diffs.warn_user_for_changes(
+        from_dir_apply_results, to_dir_apply_results
+    ):
         click.echo(warning_message, err=True)
 
-    return important_diffs.perform_diff(from_dir, to_dir)
+    important_diffs.perform_diff(from_dir, to_dir)
```

Comment:
Return statement was removed since the signature of the function is to return None